agent/scripts/newrelic.ini.template

```diff
@@ -929,6 +929,14 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ;          per transaction. A value of 0 will use the agent default.  The max
 ;          setting is 10000.
 ;
+;          !IMPORTANT: If you have Code Level Metrics enabled & long absolute
+;          pathnames to your PHP files/functions, you may exceed the max message
+;          size limit set by the Daemon if your max_samples_stored setting is
+;          too high. To fix this, you can either:
+;           a) set newrelic.code_level_metrics.enabled=false, disabling it
+;              entirely
+;           b) lower the value for newrelic.span_events.max_samples_stored
+;
```

Comment:
suggest adding:
```
c) enable infinite tracing

Consider reordering as well:
a) enable infinite tracing
b) disable clm...
c) lower the value...

Reply:
Added in f9014b84e66c84a9417641a74f53a454a2d0bf5b